src/vorta/__main__.py

```diff
@@ -5,12 +5,19 @@
 import peewee
 from vorta._version import __version__
 from vorta.config import SETTINGS_DIR
-from vorta.log import init_logger
+from vorta.log import init_logger, logger
 from vorta.models import init_db
 from vorta.updater import get_updater
 from vorta.utils import parse_args
 
 
+def exception_handler(type, value, tb):
+    # https://stackoverflow.com/questions/49065371/why-does-sys-excepthook-behave-differently-when-wrapped
+    logger.critical("Uncaught exception, file a report at https://github.com/borgbase/vorta/issues/new:",
+                    exc_info=(type, value, tb))
+    sys.__excepthook__(type, value, tb)
+
```

Comment:
If I were to add `sys.exit(1)` here, then any exception, even if it didn't cause a crash, would cause a crash.

If I don't add it here, they any exception that would cause a crash, does _not_ cause a crash.

This behaviour only appears if Vorta is in the background.

Reply:
I think I'll leave this as is, since if vorta in the background it's closable by the tray.

Reply:
Turns out the different behaviour is because I was testing with different PyQt5 versions (5.14 vs 5.15)